resources/todomvc/architecture-examples/lit-complex/package.json

```diff
@@ -12,7 +12,6 @@
     "devDependencies": {
         "big-dom-generator": "file:../../big-dom-generator",
         "http-server": "^14.1.1",
-        "jsdom": "^22.1.0",
```

Comment:
I just realized that package-lock.json wasn't regenerated after this change, would it be possible to run `npm i` in all affected projects and commit package-lock.json?

Reply:
https://github.com/WebKit/Speedometer/pull/339